course_info/static/course_info/js/controllers/IndexController.js

```diff
@@ -75,6 +90,9 @@
 
         $scope.checkIfSearchable = function() {
             $scope.searchEnabled = $scope.filtersApplied || $scope.queryString.trim() != '';
+            if (!$scope.searchEnabled) {
```

Comment:
This used to be predicated upon `searchInProgress`, not `searchEnabled`. So this means that if someone removes filters and search values, the data table disappears, even if they haven't initiated a new search?